### PR TITLE
Add resources requests

### PR DESCRIPTION
**Pull request by DReigada**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ helm delete codacy
 
 
 ## Requirements
-[All requirements](requirements/index.md)
+- [All requirements](requirements/index.md)
```
Comment (paulopontesm) [inline docs/README.md]: Do you need to split these documents?

Comment (paulopontesm) [reply docs/README.md]: I think we should start simple and putting everything in a single doc and split if necessary whenever is needed

Comment (DReigada) [reply docs/README.md]: I felt that an index file would just be a listing of all the documents in its directory. Therefore the name index.

File: docs/requirements/resources.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+### Memory
+
+`workerResources.requests.memory + (pluginResources.requests.memory * 4)`
+
+
+### Number of concurrent analysis
+`workers.genericMax + workers.dedicatedMax`
+
+### Total resources
```
[the remark this answers is not in the document]
Comment (DReigada) [reply docs/requirements/resources.md]: That would probably be a good idea, but we are still no sure on how much resources we need. @paulopontesm  any input on this?

[the remark this answers is not in the document]
Comment (DReigada) [reply docs/requirements/resources.md]: Done

**Pull request closed without merging**